### PR TITLE
Added new resource

### PR DESCRIPTION
**Pull request by haidarrifki**
Hi, i just added new resource #5

### CODE REVIEW

**Review by TenTraicion — APPROVED** [review]

File: src/assets/data/resources.json [hunk 1]
```diff
@@ -65,6 +65,16 @@
             "design"
         ]
     },
+    {
+        "title": "Prepros",
+        "description": "Compiles your files, transpiles your JavaScript, reloads your browsers and makes it really easy to develop & test your websites so you can focus on making them perfect.",
+        "image": "https://prepros.io/img/icon.png",
```
Comment (clebsonf) [inline src/assets/data/resources.json:71]: Next by default does not allow unknown image links. For safety's sake, we will continue with this approach. I released the URL user-images.githubusercontent.com, you can create an issue, attach the image to get a URL and dismiss an issue.

Comment (clebsonf) [inline src/assets/data/resources.json:71]: Next by default does not allow unknown image links. For safety's sake, we will continue with this approach. I released the url user-images.githubusercontent.com, you can create an issue, attach the image to get a url and dismiss an issue.

**Review by clebsonf — COMMENTED** [review]
Can you make this url change? From there I can accept the pull request.

**Comment by haidarrifki** [comment]
done, i just removed the image.

**Pull request merged**